packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -326,11 +329,11 @@ export const buildCompiledComponent = (opts: CompiledOpts) => {
       ? classNameProp.value.expression
       : classNameProp.value;
 
-    const newClassNameValue = t.isStringLiteral(classNameExpression)
-      ? joinExpressions(t.stringLiteral(className), classNameExpression)
-      : conditionallyJoinExpressions(t.stringLiteral(className), classNameExpression);
-
-    classNameProp.value = t.jsxExpressionContainer(newClassNameValue);
+    classNameProp.value = t.jsxExpressionContainer(
```

Comment:
this means that we conditionally apply `ax` only when we **know** it can accept outside class names.

Reply:
@pgmanutd i have to wonder if we should keep it though

if we don't what happens if someone defines a declaration like:

```
display: block;
display: flex;
```

We'd have to remove the block declaration - either at runtime or build. Builds possible though.

Reply:
🤔

Reply:
i updaated #286 with this info (see duplicate declarations)

Reply:
cool. got it. 

> let the last declaration win and then let auto prefixer take over

^^ works mate